doc/source/cost.rst

```diff
@@ -0,0 +1,45 @@
+Estimating costs
+================
+
+It is difficult to estimate costs exactly, because this depends highly on your
+particular setup. In general, there are a few primary factors that control
+cost:
+
+1. What kind of hardware do your users need?
+2. How many users do you have?
+3. What usage pattern will your users have?
+
+Hardware
+--------
+The largest driver of cost in hardware is memory (RAM). More RAM means that your
```

Comment:
~~The largest driver of cost in hardware is memory (RAM).~~
**Memory (RAM)** makes up the largest part of a cost estimate.